diagnostic.md

```diff
@@ -57,13 +73,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+   $('li').parent('body',function(){
```

Comment:
`.parent()` only returns the immediate so this will give you back the `<ol>` element.  One possible solution is `$('li').parent().parent().hide();`.